src/main/java/move/Rock.java

```diff
@@ -0,0 +1,26 @@
+package move;
+
+public class Rock implements Move {
+
+    private String name;
+
+    public Rock(String name) {
+        this.name = name;
+    }
+
+    public String playAgainst(Move opponentMove) {
+        StringBuilder verdict = new StringBuilder();
+        if (opponentMove.getName() == "scissors") {
+            verdict.append("won");
+        } else if (opponentMove.getName() == "paper"){
+            verdict.append("lost");
```

Comment:
There is a bit of duplicate between the Rock, Paper, Scissors class.

As a starting point, if the symbols changed their names, or the the verdict 'lost' and 'won' needed to be changed to 'looser' and 'winner', then think about how this could be achieved with minimal impact.